aws-sagemaker-monitoringschedule/aws-sagemaker-monitoringschedule.json

```
@@ -0,0 +1,590 @@
+          "maxLength" : 256,
+          "pattern": "^([\\p{L}\\p{Z}\\p{N}_.:/=+\\-@]*)$"
+        }
+      },
+      "required" : [ "Key", "Value" ]
+    },
+    "MonitoringExecutionSummary" : {
+      "description" : "Summary of information about monitoring job",
+      "type" : "object",
+      "properties" : {
+        "CreationTime": {
+          "description": "The time at which the monitoring job was created.",
+          "type": "string"
+        },
+        "EndpointName": {
+          "$ref" : "#/definitions/EndpointName"
+        },
+        "FailureReason" : {
+          "type" : "string",
+          "description" : "Contains the reason a monitoring job failed, if it failed.",
+          "maxLength" : 1024
+        },
+        "LastModifiedTime": {
+          "description": "A timestamp that indicates the last time the monitoring job was modified.",
+          "type": "string"
+        },
+        "MonitoringExecutionStatus": {
+          "description": "The status of the monitoring job.",
+          "type": "string",
+          "enum": [
+            "Pending",
+            "Completed",
+            "CompletedWithViolations",
+            "InProgress",
+            "Failed",
+            "Stopping",
+            "Stopped"
+          ]
+        },
+        "MonitoringScheduleName": {
+          "$ref" : "#/definitions/MonitoringScheduleName"
+        },
+        "ProcessingJobArn" : {
+          "description": "The Amazon Resource Name (ARN) of the monitoring job.",
+          "type" : "string",
+          "pattern": "aws[a-z\\-]*:sagemaker:[a-z0-9\\-]*:[0-9]{12}:processing-job/.*",
+          "maxLength": 256
+        },
+        "ScheduledTime": {
+          "description": "The time the monitoring job was scheduled.",
+          "type": "string"
+        }
+      },
+      "required" : [ "CreationTime", "LastModifiedTime", "MonitoringExecutionStatus", "MonitoringScheduleName", "ScheduledTime" ]
+    },
+    "EndpointName": {
+      "type" : "string",
+      "description" : "The name of the endpoint used to run the monitoring job.",
+      "pattern": "^[a-zA-Z0-9](-*[a-zA-Z0-9])*",
+      "maxLength" : 63
+    },
+    "MonitoringScheduleName": {
+      "type" : "string",
+      "description" : "The name of the monitoring schedule.",
+      "pattern": "^[a-zA-Z0-9](-*[a-zA-Z0-9])*$",
+      "maxLength" : 63
+    }
+  },
+  "required" : [ "MonitoringScheduleConfig", "MonitoringScheduleName" ],
+  "primaryIdentifier" : [ "/properties/MonitoringScheduleArn" ],
+  "handlers": {
+    "create": {
+      "permissions": [
+        "sagemaker:CreateMonitoringSchedule",
+        "sagemaker:DescribeMonitoringSchedule",
+        "iam:PassRole"
+      ]
+    },
+    "delete": {
+      "permissions": [
+        "sagemaker:DeleteMonitoringSchedule",
+        "sagemaker:DescribeMonitoringSchedule"
+      ]
+    },
+    "list": {
+      "permissions": [
+        "sagemaker:ListMonitoringSchedule"
```

```suggestion
        "sagemaker:ListMonitoringSchedules"
```
https://docs.aws.amazon.com/service-authorization/latest/reference/list_amazonsagemaker.html